.vscode/launch.json

```diff
@@ -10,7 +10,7 @@
             "request": "launch",
             "preLaunchTask": "build",
             // If you have changed target frameworks, make sure to update the program path.
-            "program": "${workspaceFolder}/osu.Game.Rulesets.Diva.Tests/bin/Debug/netcoreapp3.1/osu.Game.Rulesets.Diva.Tests.dll",
+            "program": "${workspaceFolder}/osu.Game.Rulesets.Diva.Tests/bin/Debug/net5.0/osu.Game.Rulesets.Diva.Tests.dll",
```

Comment:
Do we really need to push `.vscode`? It's not important for the project

Reply:
its not necessary but still useful